auditexport/cli.py

```diff
@@ -117,40 +124,46 @@ def process(args, auditclient, utils):
         'method': 'GET',
         'query': {
             'start': queryStart,
-            'end': queryEnd
+            'end': queryEnd,
+            'sort': 'timestamp:asc',
         }
     }
 
-    if(nextPageStartKey and useBookMark):
-        req['query']['nextPageStartKey'] = nextPageStartKey
+    if(nextPageCursor and useCursor):
+        req['query']['nextPageCursor'] = nextPageCursor
+
+    if(useCursor):
```

Comment:
should this be `if(limit)`?

Reply:
Yeah. Probably it should be `if(limit or cursor)` cause if we use cursor we should set limit (or we won't split request on chunks)